tests/inspect/inspect.go

```diff
@@ -57,6 +59,8 @@ func init() {
 	globalFlagset.BoolVar(&globalFlags.ReadFile, "read-file", false, "Print the content of the file $FILE")
 	globalFlagset.BoolVar(&globalFlags.WriteFile, "write-file", false, "Write $CONTENT in the file $FILE")
 	globalFlagset.IntVar(&globalFlags.Sleep, "sleep", -1, "Sleep before exiting (in seconds)")
+	globalFlagset.StringVar(&globalFlags.FileName, "file-name", "", "The file to read/write, $FILE will be ignored if this is specified")
```

Comment:
@jonboulle 
If we use the environment, then the podManifest.Apps[i].App will be non-nil, and will shadow the imageManifest.App.
In some test cases(https://github.com/coreos/rkt/pull/980/files#diff-000db758bcc3d62d3fb677f45685d25aR274) I need to test when the pod manifest.App does not shadow the imageManifest.App, so I need to provide this argument to the App.exec to get the filename. Does that make sense? Or do you have better suggestions?